### PR TITLE
Update jquery.grideditor.js

### PR DESCRIPTION
**Pull request by atzeteppema**
put id of row or col in html code
see issue #49

### CODE REVIEW

File: src/js/jquery.grideditor.js [hunk 1]
```diff
@@ -353,12 +353,15 @@ $.fn.gridEditor = function( options ) {
         function createDetails(container, cssClasses) {
             var detailsDiv = $('<div class="ge-details" />');
 
-            $('<input class="ge-id" />')
+            var input = $('<input class="ge-id" />')
```
Comment (Epskampie) [inline src/js/jquery.grideditor.js]: No need to give it a name

File: src/js/jquery.grideditor.js [hunk 2]
```diff
@@ -353,12 +353,15 @@ $.fn.gridEditor = function( options ) {
...
                 .attr('placeholder', 'id')
                 .val(container.attr('id'))
                 .attr('title', 'Set a unique identifier')
                 .appendTo(detailsDiv)
             ;
+            input.change(function () {
+                container.attr('id', this.value);
```
Comment (Epskampie) [inline src/js/jquery.grideditor.js]: Please move inline with the other calls above (around line #359)

File: src/js/jquery.grideditor.js [hunk 3]
```diff
@@ -556,4 +559,4 @@ $.fn.gridEditor = function( options ) {
 
 $.fn.gridEditor.RTEs = {};
 
-})( jQuery );
```
Comment (Epskampie) [inline src/js/jquery.grideditor.js:559]: Please revert this irrelevant change.

**Review by Epskampie — CHANGES_REQUESTED** [review]
Thanks, i've added some remarks.

**Comment by atzeteppema** [comment]
Thanks for the remarks.
I made the changes with the Github webinterface.
Can you teach me how to revert a comment there? I can’t find it.

Since the webinterface of GitHub adds the blank ending line at the file, I will redo my changes with my code editor and try again.

Best regards,
Atze Teppema

Van: Epskampie [mailto:notifications@github.com]
Verzonden: vrijdag 14 juli 2017 08:03
Aan: Frontwise/grid-editor <grid-editor@noreply.github.com>
CC: Atze Teppema <atze.teppema@tttsoftware.nl>; Author <author@noreply.github.com>
Onderwerp: Re: [Frontwise/grid-editor] Update jquery.grideditor.js (#53)


@Epskampie requested changes on this pull request.

Thanks, i've added some remarks.

________________________________

In src/js/jquery.grideditor.js<https://github.com/Frontwise/grid-editor/pull/53#discussion_r127389722>:

> @@ -353,12 +353,15 @@ $.fn.gridEditor = function( options ) {

         function createDetails(container, cssClasses) {

             var detailsDiv = $('<div class="ge-details" />');



-            $('<input class="ge-id" />')

+            var input = $('<input class="ge-id" />')

No need to give it a name

________________________________

In src/js/jquery.grideditor.js<https://github.com/Frontwise/grid-editor/pull/53#discussion_r127389863>:

>                  .attr('placeholder', 'id')

                 .val(container.attr('id'))

                 .attr('title', 'Set a unique identifier')

                 .appendTo(detailsDiv)

             ;

+            input.change(function () {

+                container.attr('id', this.value);

Please move inline with the other calls above (around line #359)

________________________________

In src/js/jquery.grideditor.js<https://github.com/Frontwise/grid-editor/pull/53#discussion_r127389910>:

> @@ -556,4 +559,4 @@ $.fn.gridEditor = function( options ) {



 $.fn.gridEditor.RTEs = {};



-})( jQuery );

Please revert this irrelevant change.

**Comment by Epskampie** [comment]
Changes were moved to #54

**Pull request closed without merging**